spec/services/stripe_model_callbacks/invoice/updated_service_spec.rb

```diff
@@ -0,0 +1,65 @@
+require "rails_helper"
+
+describe StripeModelCallbacks::Invoice::UpdatedService do
+  subject(:updated_service) { described_class.execute!(event: stripe_event) }
```

Comment:
1. I am moving away from using `described_class` :-) 
2. This is called `updated_service` but doesn't return a service. It returns a result from the service (whatever that might be?)
```suggestion
  subject(:updated_service_result) { StripeModelCallbacks::Invoice::UpdatedService.execute!(event: stripe_event) }
```

Reply:
fixed